lib/internal/main/repl.js

```diff
@@ -19,38 +19,44 @@ prepareMainThreadExecution();
 
 markBootstrapComplete();
 
-// --input-type flag not supported in REPL
-if (getOptionValue('--input-type')) {
-  // If we can't write to stderr, we'd like to make this a noop,
-  // so use console.error.
-  console.error('Cannot specify --input-type for REPL');
-  process.exit(1);
-}
+if (process.env.NODE_REPL_EXTERNAL_MODULE) {
+  require('internal/modules/cjs/loader')
+    .Module
+    ._load(process.env.NODE_REPL_EXTERNAL_MODULE, undefined, true);
```

Comment:
Would adding a `return` here work? If so, the `else` could go away, and the diff would be a lot simpler.

Reply:
It would, but I try to avoid top-level return.